src/App.js

```diff
@@ -38,121 +38,125 @@ function App() {
       </div>
       <div className="border-2 border-solid border-black grid text-3xl grid-cols-4 grid-rows-5 divide-x-2 divide-y-2 w-[32rem] h-[44rem]">
         <button
-          className="h-full w-full bg-orange-500 font-bold text-white"
+          className="h-full w-full bg-orange-500 col-span-2 hover:bg-orange-600 transition-all font-bold text-white"
           onClick={(e) => setExpression("")}
         >
           AC
         </button>
-        <button
+        {/* TODO: for later implementation */}
+        {/* <button
           onClick={(e) => addToExpression(e.target.textContent)}
-          className="h-full w-full bg-orange-500 font-bold text-white"
+          className="h-full w-full bg-orange-500 hover:bg-red-600 transition-all font-bold text-white"
         >
           +/-
-        </button>
+        </button> */}
         <button
           onClick={(e) => addToExpression(e.target.textContent)}
-          className="h-full w-full bg-orange-500 font-bold text-white"
+          className="h-full w-full bg-orange-500 hover:bg-orange-600 transition-all font-bold text-white"
```

Comment:
can you create class in our app.css and apply this tailwind code inside. Because you have use it multiple time and it will be good if you have as one classname.